main.go

```diff
@@ -135,13 +153,19 @@ func main() {
 		logger:  logger,
 	}
 
+	if *smartctlRescanInterval > 1*time.Second && len(*smartctlDevices) > 0 {
```

Comment:
Should be greater or equal `>= 0`, because when devices are not defined is a zero len

Reply:
You're right. I think it should be `== 0`. So if any devices are configured via the the `smartctl.device` option, if they exist or not, the background task should not run. Am I right?

Reply:
Seems right